praw/models/reddit/subreddit.py

```diff
@@ -1776,6 +1854,13 @@ def revisions(self, **generator_kwargs):
         Additional keyword arguments are passed in the initialization of
         :class:`.ListingGenerator`.
 
+        To view the wiki revisions for ``'praw_test'`` in ``'/r/test'`` try:
+
+        .. code:: python
+
+          for item in reddit.subreddit('test').wiki['praw_test'].revisions():
```

Comment:
That would put it over the 80 characters per line. Would something like this work:

        .. code:: python

           for revision in (reddit.subreddit('test').wiki['praw_test']
                                                    .revisions()):
               print(revision)